lib/trento/domain/value_objects/saptune_staging.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Trento.Domain.SaptuneStaging do
+  @required_fields [:id]
```

Comment:
This should be something else, `[:enabled]` maybe?

Reply:
nice catch, was c&p